docs/user/document/distribution/tlp/url-usage.en.md

```diff
@@ -1 +1,7 @@
 # Url - Usage
+
+Reference to the meaning of the used TLP labels.
+The default value should be a link to the schema at first.org.
```

Comment:
```suggestion
The default value is a link to the definitions from FIRST.
```
Add link here.

Reply:
Added a link to https://www.first.org/tlp/ - is that what you meant?